methods/id.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "id",
+    "status": "registered",
+    "verifiableDataRegistry": "ID Service",
+    "contactName": "Mastercard",
+    "contactEmail": "",
+    "contactWebsite": "https://idservice.com/en/contactus.html",
```

Comment:
Is this the best way to reach out in the event the WG needs to request something from the method authors?

Reply:
I like the idea of using a website URL to contact the authors. I suspect that 95% of the registrations will use that instead of supplying an email.

Reply:
Yeah I'm fine with it too. Just wanted to confirm an email wasn't a better way because that web link looks to be more generic in its purpose for contacting their team

Reply:
This form gets you in direct contact with our product team.  They can divert your message internally as necessary.